src/MasterSite.php

```diff
@@ -279,6 +279,37 @@ public function set_featured_image( $post_id, $post, $update ) {
 		}
 	}
 
+	/**
+	 * Generates hreflang metadata from countries.json template.
+	 */
+	public function generate_hreflang_meta() {
+
+		$request   = wp_remote_get( $this->theme_dir . '/templates/countries.json' );
+		$body      = wp_remote_retrieve_body( $request );
+		$countries = json_decode( $body, true );
+
+		$metadata = '';
+
+		foreach ( $countries as $key => $letter ) {
+			if ( 0 === $key ) {
+				$metadata .= "<link rel='alternate' hreflang='x-default' href='" . $letter[ $key ]['url'] . "'>\n";
+				continue;
+			}
+			foreach ( $letter as $country ) {
```

Comment:
It is unfortunately :) Because then we would have to refactor the js code on both the country selector and the landing page. Maybe after this is done we could re-think whether it makes sense to render the country selector in the backend.